docs/arf.md

```diff
@@ -1829,7 +1844,7 @@ established:
 
 3.  The EUDI Wallet Instance authenticates the EUDI Wallet Provider,
```

Comment:
how does the instance authenticate the wallet provider? based on the signature and the trust chain in the WTE?

Reply:
probably using a hardcoded certificate/TLS certificate pinning, but that's not relevant for interop